spec/features/system/admin/instance_management_spec.rb

```diff
@@ -48,8 +48,10 @@
       end
 
       scenario 'I can see all instances' do
-        instances = create_list(:instance, 2)
-        visit admin_instances_path(page: Instance.page.total_pages)
+        create_list(:instance, 2)
+        last_page = Instance.page.total_pages
+        instances = Instance.order_by_id.page(last_page)
```

Comment:
this spec is a bit different now. previously: create 2 instances, check that they appear on the page. now: create 2 instances, check that the last page appears... is that sufficient?